index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.0/css/bootstrap.min.css">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
+</head>
+<body>
+ 
+    <header>
+        <nav>
+         <a href="/">My Logo</a>
+         <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="w-6 h-6">
+            <path stroke-linecap="round" stroke-linejoin="round" d="M3.75 6.75h16.5M3.75 12h16.5m-16.5 5.25h16.5" />
+          </svg>
+          
+        </nav> 
+    </header> 
+    <main>
+        <section class="intro">
+            <h1>I’m Okeke Okechukwu.
+                    Glad to see you!</h2>
```

Comment:
- [ ] You have executed the recommended modifications exceptionally well, but it is important to ensure that each element you utilize maintains consistent enclosing elements. For instance, the `<h1>` tag should also be accompanied by the closing `</h1>` tag.